backend/core/domains/messaging/consumers.py

```
@@ -0,0 +1,389 @@
+            elif message_type == 'typing_indicator':
+                await self.handle_typing_indicator(data)
+            elif message_type == 'mark_as_read':
+                await self.handle_mark_as_read(data)
+            elif message_type == 'ping':
+                await self.send(text_data=json.dumps({'type': 'pong'}))
+            else:
+                await self.send_error('Unknown message type')
+
+        except json.JSONDecodeError:
+            await self.send_error('Invalid JSON format')
+        except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)